src/index.js

```diff
@@ -2,11 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import ShoppingCar from './shoppingCar/Car';
 import * as serviceWorker from './serviceWorker';
+import {BrowserRouter, Routes, Route} from 'react-router-dom'
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Routes>
+        <Route path='/' element={<ShoppingCar/>}/>
+        <Route path='/newcard' element={<App />}/>
```

Comment:
```suggestion
        <Route path='/payment' element={<App />}/>
```

Notice it refers to an action, to pay